Coverless/Coverless/DesignSystem/Protocols/LabelDesignable.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LabelDesignable.swift
+//  Coverless
+//
+//  Created by Gabriel Ferreira de Carvalho on 15/09/21.
+//
+
+import UIKit
+
+protocol LabelDesignable {
+    func stylize(with textStyle: TextStyle)
+    func setupAcessibility()
+}
+
+extension UILabel: LabelDesignable {
```

Comment:
Protocol e Extension estão na mesma pasta, mesmo sendo do mesmo assunto seria bom deixar em arquivos separados.

Reply:
Vou corrigir isso no próximo PR, isso ta meio errado to achando